platform/darwin/src/MGLLocationManager.h

```diff
@@ -0,0 +1,175 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLLocationManagerDelegate;
+
+/**
+ The `MGLLocationManager` protocol defines a set of methods that a class must
+ implement in order to serve as the location manager of an `MGLMapView`. A location
+ manager is responsible for notifying the map view about location-related events,
+ such as a change in the user’s location. This protocol is similar to the
+ Core Location framework’s `CLLocationManager` class, but your implementation
+ does not need to be based on `CLLocationManager`.
+ 
+ */
+@protocol MGLLocationManager <NSObject>
+
+@optional
+
+#pragma mark Determining Location Updates Accuracy
```

Comment:
Nit: I wouldn’t say this section is about “determining” the accuracy — it’s probably more accurate to say that this section is about “Configuring Location Update Precision”.